template-package/replaceMe.test.ts

```diff
@@ -1,7 +1,9 @@
-import replaceMe from './replaceMe'
+import assert from 'node:assert'
+import { describe, test } from 'node:test'
```

Comment:
Vitest

Reply:
This repo uses `node --test`, not Vitest.

Reply:
Ohh 😵